src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angu
 import { FormBuilder, FormGroup } from '@angular/forms';
 import { Subscription } from 'rxjs';
 import { ItemType } from '../../../../shared/helpers/item-type';
+import { ChildData } from '../../../item/components/item-children-edit/item-children-edit.component';
```

Comment:
You cannot make a component depends on a parent component.

Reply:
Fixed